samples/ingestion/ingestion-client/Setup/ArmTemplateBatch.json

```diff
@@ -115,6 +115,15 @@
                 "description": "Enter the address of your private endpoint here (e.g. https://mycustomendpoint.cognitiveservices.azure.com/) if you are connecting with a private endpoint"
             }
         },
+        "StartTranscriptionFunctionTimeInterval":
+        {
+            "defaultValue": "0 */2 * * * *",
```

Comment:
I think we'd like to only ask for a numeric value of the timer interval since the users can be non-technical. So basically just accept the interval and generate the cron string programmatically.

Reply:
Based on @komalg1's research, Azure Functions don't support dynamic cron expressions in the trigger attribute anyway.

So the choice comes down to exposing either a cron or a TimeSpan directly to the users as a parameter. While cron expressions allow more flexibility, TimeSpan is easier for non-technical users to understand. Given that IngestionClient is promoted as a no-code solution, we need to be careful about this tradeoff between ease of use vs flexibilty (keeping in mind the only flexibility that TimeSpan doesn't seem to provide as compared to Cron is obscure cases like day-of-month, year etc which might not be relevant in the case of the StartTranscription function). 

@HenryvanderVegte - would you like to chime in?

Reply:
There is no code change required if we want to use TimeSpan. Only condition is that the function needs to be running on an App Service Plan if TimeSpan has to be used as TimeSpan does not work with Consumption Plan.
Let me know your thoughts.

Reply:
Received a comment offline from Henry: "If TimeSpan is not supported by Consumption Plan, I think it's fine to keep CRON expressions. Still supporting Consumption is more important I believe." So let's keep the change as it is for now. Thanks Komal for bearing with me :)